src/tests/decider-distro-tests-rs/tests/test.rs

```diff
@@ -119,3 +533,1006 @@ async fn test_decider_installed() {
         assert!(names.contains(&alias2), "{alias2} service is not installed");
     }
 }
+/// Test the basic flow
+///
+/// 1. *Decider* asks the last block of the chain from which to start polling
+///    The block number is `0x10`
+///
+/// 2. *Decider* asks for the logs from the block `0x10` to the `0x10 + 500` blocks (range configured in the connector)
+///    We return the logs with pre-defined CID of the url-downloader app. This step required IPFS node with the app.
```

Comment:
What does "requires IPFS node with the app" mean?

I'd read it as `"Downloading this CID requires an available IPFS instance"`. Is that correct?

```suggestion
///    We return the logs with pre-defined CID of the url-downloader app. This step requires IPFS node with the app.
```

Reply:
or does "with the app" mean `"an IPFS instance with url-downloader app package pre-uploaded to it"`?

Reply:
Yep. It means that in CI we need to have IPFS node with the specific uploaded services.